src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -4937,4 +4961,66 @@ export class PostgresCohortMembersService {
       throw error;
     }
   }
+
+  /**
+   * Builds WHERE clause for searchtext filtering across multiple columns
+   * @param searchtext - The search text to filter by
+   * @returns SQL WHERE clause string for searchtext filtering
+   */
+  private buildSearchTextWhereClause(searchtext: string): string {
+    try {
+      if (!searchtext || typeof searchtext !== 'string' || searchtext.trim().length < 2) {
```

ESLint

[error] Replace `!searchtext·||·typeof·searchtext·!==·'string'·||·searchtext.trim().length·<·2` with `⏎········!searchtext·||⏎········typeof·searchtext·!==·"string"·||⏎········searchtext.trim().length·<·2⏎······`

(prettier/prettier)